docs/tutorial-osparc.ipynb

```diff
@@ -0,0 +1,437 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# o²S²PARC module from the NIH SPARC Python Client\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this document you will learn how to access the [o²S²PARC Python Client](https://itisfoundation.github.io/osparc-simcore-clients/#/) as part fo the [NIH SPARC Python Client](https://github.com/nih-sparc/sparc.client).\n",
+    "\n",
+    "### Pre-requites:\n",
+    "- Python version >=3.10: as requested by the [sparc.client](https://pypi.org/project/sparc.client/) package.\n",
+    "- An account on [osparc.io](https://osparc.io): you will need this to generate API tokens to use the o²S²PARC module. If you don't have an account, request one at oSPARC Support, as explained [here](https://sparc.science/resources/4LkLiH5s4FV0LVJd3htsvH).\n",
+    "- Have a look to the [NIH SPARC Python Client doc](https://github.com/nih-sparc/sparc.client/blob/main/docs/tutorial.ipynb) as a preliminary step (and learn how to set up the Configuration)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Installation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/iavarone/.pyenv/versions/3.10.11/envs/sparc_client/lib/python3.10/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
+      "  from .autonotebook import tqdm as notebook_tqdm\n"
+     ]
+    }
+   ],
+   "source": [
+    "# import the necessary packages\n",
+    "import importlib, getpass\n",
+    "# `is None: # This give and error (cannot find sparc)\n",
+    "# ! pip install sparc.client\n",
+    "try:\n",
+    "  import sparc.client\n",
+    "except ImportError:\n",
+    "  ! pip install git+https://github.com/nih-sparc/sparc.client\n",
```

Comment:
We don't want to be installing from the GitHub repository, we only want official releases.

Reply:
Yep, thanks for pointing this out. This was here because I had to install from github because `oSparc` part of the client isn't available in the latest release yet